CHANGELOG.md

```diff
@@ -761,4 +755,4 @@ This release contains bug fixes and AIO path changes.
 [2.1.5]: https://github.com/puppetlabs/ezbake/compare/2.1.4...2.1.5
 [2.1.4]: https://github.com/puppetlabs/ezbake/compare/2.1.3...2.1.4
 [2.1.3]: https://github.com/puppetlabs/ezbake/compare/2.1.2...2.1.3
-[2.1.2]: https://github.com/puppetlabs/ezbake/compare/2.1.1...2.1.2
\ No newline at end of file
+[2.1.2]: https://github.com/puppetlabs/ezbake/compare/2.1.1...2.1.2
```

Comment:
Should this be removing 2.1.6?

Reply:
I think this is one of those weird newline changes, but yeah, the 2.1.6 line should also be removed, good catch. not sure why the revert didn't get that..

Reply:
ahh, because we used to not have links, so that's a separate commit. updating now.